app/models/user.rb

```diff
@@ -4,6 +4,8 @@ class User < ApplicationRecord
   has_many :likes
   has_many :liked_posts, through: :likes, source: :post
 
+  validates :name, presence: true
+
```

Comment:
- [ ] Good job so far, Kindly add validation for your Posts Counter in your User model, from the requirement; `PostsCounter must be an integer greater than or equal to zero`, 


`